internal/git/git.go

```diff
@@ -18,45 +18,50 @@ package git
 import (
 	"bytes"
 	"context"
-	"crypto/x509"
 	"fmt"
 
 	"github.com/sigstore/gitsign/internal/fulcio"
 	"github.com/sigstore/gitsign/internal/signature"
 	"github.com/sigstore/gitsign/pkg/git"
 	"github.com/sigstore/gitsign/pkg/rekor"
-	"github.com/sigstore/rekor/pkg/generated/models"
 )
 
-func Sign(ctx context.Context, rekor rekor.Writer, ident *fulcio.Identity, data []byte, opts signature.SignOptions) ([]byte, *x509.Certificate, *models.LogEntryAnon, error) {
-	sig, cert, err := signature.Sign(ident, data, opts)
+func Sign(ctx context.Context, rekor rekor.Writer, ident *fulcio.Identity, data []byte, opts signature.SignOptions) (*signature.SignResponse, error) {
+	resp, err := signature.Sign(ctx, ident, data, opts)
 	if err != nil {
-		return nil, nil, nil, fmt.Errorf("failed to sign message: %w", err)
+		return nil, fmt.Errorf("failed to sign message: %w", err)
 	}
 
+	// We're using offline verification style signing - nothing more to do.
```

Comment:
Note to self (maybe worth inlining in the comment): nothing more to do *because `signature.Sign` already did it*

Reply:
Shouldn't be a problem with the refactor.